### PR TITLE
Bring additions from my fork over to main.

### PR DESCRIPTION
**Pull request by Siuolplex**
Adds a couple of additional features and goodies from my fork over to this.
It aint much, but I think it should help this mod a bit.

Most notably, this commit adds Quilt.Mod.Json, a crash limit amount thing in config, and fixes a small error.

BTW, I credit myself as a contributor in Quilt.mod.json but not Fabric.mod.json. Feel free to change that if you want.

### CODE REVIEW

File: common/src/main/java/fudge/notenoughcrashes/stacktrace/CrashUtils.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 public final class CrashUtils {
 
-    private static final Logger LOGGER = LogManager.getLogger("TMC");
+    private static final Logger LOGGER = LogManager.getLogger("NEC");
```
Comment (natanfudge) [inline common/src/main/java/fudge/notenoughcrashes/stacktrace/CrashUtils.java:16]: oh ok lol

File: common/src/main/java/fudge/notenoughcrashes/NecConfig.java [hunk 2]
```diff
@@ -16,6 +16,7 @@
     public boolean catchInitializationCrashes = true;
     public boolean debugModIdentification = false;
     public boolean forceCrashScreen = false;
+    public int crashLimit = 20;
```
Comment (natanfudge) [inline common/src/main/java/fudge/notenoughcrashes/NecConfig.java:19]: yeah that makes sense

File: common/src/main/java/fudge/notenoughcrashes/gui/ProblemScreen.java [hunk 3]
```diff
@@ -56,7 +56,7 @@ protected ProblemScreen(CrashReport report) {
     }
 
 
-    private static final Set<String> IGNORED_MODS = new HashSet<>(Arrays.asList("minecraft", "fabricloader", "loadcatcher", "jumploader"));
+    private static final Set<String> IGNORED_MODS = new HashSet<>(Arrays.asList("minecraft", "fabricloader", "loadcatcher", "jumploader", "quilt_loader"));
```
Comment (natanfudge) [inline common/src/main/java/fudge/notenoughcrashes/gui/ProblemScreen.java:59]: nice

File: fabric/src/main/resources/fabric.mod.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "schemaVersion": 1,
   "id": "notenoughcrashes",
   "name": "Not Enough Crashes",
-  "description": "A fork of TooManyCrashes, a port of VanillaFix's crash improvement features to Fabric, for Minecraft 1.15+",
+  "description": "A fork of TooManyCrashes, a port of VanillaFix's crash improvement features to Fabric and Quilt mod loaders, for Minecraft 1.15+",
```
Comment (natanfudge) [inline fabric/src/main/resources/fabric.mod.json]:
```suggestion
  "description": "Improves crashes in Minecraft - allows returning to title screen, blaming causing mods, and more.",
```

File: fabric/src/main/resources/quilt.mod.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    },
+    "depends": [
+      {
+        "id": "minecraft",
+        "versions": ">=1.17.1"
+      }
+    ],
+    "metadata": {
+      "name": "Not Enough Crashes",
+      "description": "A fork of TooManyCrashes, a port of VanillaFix's crash improvement features to Fabric and Quilt mod loaders, for Minecraft 1.15+",
```
Comment (natanfudge) [inline fabric/src/main/resources/quilt.mod.json]: Update this with my suggested change

File: fabric/src/main/resources/quilt.mod.json [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+      "contributors": {
+        "Fudge": "NEC Dev",
```
Comment (natanfudge) [inline fabric/src/main/resources/quilt.mod.json]:
```suggestion
        "Fudge": "NEC Maintainer",
```

**Review by natanfudge — CHANGES_REQUESTED** [review]
LGTM after changes.

File: common/src/main/java/fudge/notenoughcrashes/stacktrace/CrashUtils.java (hunk 1 above)
Comment (natanfudge) [inline common/src/main/java/fudge/notenoughcrashes/stacktrace/CrashUtils.java]: Delete this logger and use `NotEnoughCrashes.LOGGER` instead

Comment (Siuolplex) [reply common/src/main/java/fudge/notenoughcrashes/stacktrace/CrashUtils.java]: Fixed!

**Comment by Siuolplex** [comment]
Added Github Actions as well, because its useful.

**Comment by natanfudge** [comment]
Thanks!

**Pull request merged**